src/crystal/system/unix/time.cr

```diff
@@ -14,7 +14,7 @@ module Crystal::System::Time
   UnixEpochInSeconds = 62135596800_i64
 
   def self.compute_utc_seconds_and_nanoseconds : {Int64, Int32}
-    {% if LibC.methods.includes?("clock_gettime".id) %}
+    {% if LibC.methods.any? &.name.==("clock_gettime") %}
```

Comment:
Does  `LibC.has_method?`  work?

Reply:
Wow, there's such thing. This language is great!